rollup.config.js

```diff
@@ -27,11 +27,18 @@ export default {
     console.warn(warning.message);
   },
   output: {
-    sourcemap: true,
+    entryFileNames: (chunkInfo) => {
+      // Don't include `svelte` in filename
+      return `${chunkInfo.name}.js`;
+    },
+    compact: true,
+    dir: 'package/',
     format: 'es',
     name: 'login-widget',
-    dir: 'package/',
     plugins: [],
+    preserveModules: true,
```

Comment:
Why do we want to preserve modules?

Reply:
For anyone reading this, Ryan and I discussed how I'm currently preserving modules for easier debugging in external apps. We have a story to finalize how we want to bundle this here: https://bugster.forgerock.org/jira/browse/SDKS-2082